app/modules/asset_groups/models.py

```diff
@@ -457,7 +466,7 @@ def detected(self, job_id, response):
 
             for annot_id in range(len(results)):
                 annot_data = results[annot_id]
-                annot_uuid = annot_data.get('uuid', {}).get('__UUID__')
+                annot_uuid = annot_data.get('uuid', {})
```

Comment:
weird, i am still seeing the `{ "__UUID__": "x-x-x-x" }` format comming from detection results.

Reply:
This is what I really am not clear about. It looks like historically I wrote detection responses with both formats in. I really don't know which is correct. I don't know if this has changed recently. I guess the real way is to get a response from Sage to one of these requests and look at it